packages/modal/stories.tsx

```diff
@@ -95,7 +95,7 @@ storiesOf('Modal Container', module)
               getTitleProps,
               getContentProps,
               getCloseProps
-            }) => {
+            }: IUseModalReturnValue) => {
```

Comment:
If `ModalContainer` is typed correct we shouldn't need to cast this as the return value

Reply:
Updated, thanks!